web/src/Message.tsx

```diff
@@ -0,0 +1,59 @@
+import cx from "classnames";
+import css from "./Message.module.css";
+
+const appOrigin = "Riley";
+
+interface MessageProperties {
+  fromUser: string;
+  contents: string;
+  optimistic: boolean;
+  isLoading?: boolean;
+  error?: unknown;
+}
+
+function Message({
```

Comment:
Where is this component used?

Reply:
this component doesn't get used anymore. I believe it is what was being used prior to my changes.

Reply:
Then let's remove it. No sense in keeping around code that doesn't get used.